lib/http.js

```diff
@@ -123,7 +124,11 @@ module.exports = {
   validateHeaderName,
   validateHeaderValue,
   get,
-  request
+  request,
+  setMaxIdleHTTPParsers(max) {
+    validateInteger(max, 'max');
```

Comment:
I don't think we should accept negative number right?
```suggestion
    validateInteger(max, 'max', 1);
```

Reply:
Done !